docs/alerting/README.md

```diff
@@ -48,8 +48,8 @@ Other pai component also used some metrics for display, they are:
     <td>
         <ul>
             <li>node_uname_info</li>
-            <li>nvidiasmi_utilization_gpu</li>
-            <li>nvidiasmi_utilization_memory</li>
+            <li>gpu_utilization</li>
+            <li>gpu_mem_utilization</li>
```

Comment:
I prefer the name **`gpu_util`** and **`memory_usage`**, according to pretty printed `nvidia-smi` output.
util is a percentage, while usage is a value of XX MB, there's also a `memory_util` field in smi xml output.

Reply:
Here `gpu_mem_utilization` is a percentage. In Grafana, we only show the percentage of GPU memory utilization, not the usage values